book/website/collaboration/inclusive-events/inclusive-events-location.md

```diff
@@ -0,0 +1,249 @@
+## Considering Transport Links
+
+When planning an inclusive event for people with lived experience, careful consideration of transportation is essential to ensure that all attendees will be able to access the venue comfortably and safely.
+
+As part of resource planning and depending on the resources available, you may want to consider allocating budget related to the Accessibility fund that can be dedicated to addressing transportation needs. 
+For example, supporting bursaries that cover train tickets and travel for carers and support workers who may need to travel alongside an attendee. 
+
+You should also consider what other access requirements your attendees may require. 
+A multi-day event may involve a social dinner at a separate location which would require the organization of accessible transport options, shuttle services, or potentially a subsidy for attendees with mobility challenges who arrange their own transportation. 
+
+The provision of sufficient parking, both accessible and standard, is important as some participants may prefer travelling by car due to mobility impairments. 
+Try to ensure that there is adequate parking so that all attendees who wish to travel by car are able to park at no disadvantage. 
+You may want to consider if your event can reimburse attendees who need to access paid parking if there are no designated parking spaces at the venue. 
+
+In addition to sufficient parking, when visiting a venue try to check if both parking lots, and the paths to access any parking or transport links, are well-lit. 
+Proper lighting reduces the risk of falls or injury in the dark and enhances safety and helps individuals navigate comfortably, particularly for those who may have visual impairments or mobility difficulties. 
+If your event does start or end outside of daylight hours, try to organize transport or a walking group that guides people directly to and from the venue so that no one is travelling alone. 
+
+Events such as networking and formal dinners that continue into the evening should prioritize having well-lit pathways and common areas to ensure that attendees can move about comfortably and safely. 
+
+Finally, organizers should consider the availability of taxi services in the local area close to the event location. 
+If possible, designate a member of staff or organizer to be responsible for assisting attendees in arranging taxi bookings who will be available in the lead-up to the event, 
+and also available on the day to ensure that all booked taxis have arrived. 
+Having a point of contact person who can facilitate transportation arrangements can be invaluable, especially for people with lived experience who rely on taxi services due to mobility impairments. 
+
+By addressing any barriers that transportation access may cause, event organizers can ensure that attendees with lived experience feel welcomed and are able to engage fully in the event.
+
+
+(cl-inclusive-events-catering)=
+## Catering
```

Comment:
you probably already did but on the off-chance: Worth checking whether this replicates any of the general event planning things, as food allergies are so common!

Reply:
Okay, perfect! Then we can for now just keep it all in here and link from the event organisation page!